frontend/src/pages/StudentsPage/StudentDetails.jsx

```diff
@@ -0,0 +1,136 @@
+import { Link, useParams } from "react-router-dom";
+import STUDENTS from "../../prototypeData/students";
+import Button from "react-bootstrap/Button";
+import Col from "react-bootstrap/Col";
+import Form from "react-bootstrap/Form";
+import Row from "react-bootstrap/Row";
+import FACULTIES from "../../prototypeData/faculties";
+import Container from "react-bootstrap/Container";
+
+const StudentDetails = () => {
+  const { vidko } = useParams();
+  const user = STUDENTS.find((student) => student.vidko === vidko);
+
+  return (
+    <>
+      <h1>Studento duomenys ({vidko})</h1>
+      <Container>
+        <Form>
+          <Row>
+            <Col>
+              <Form.Group className="mb-3" controlId="Firstname">
+                <Form.Label>Vardas</Form.Label>
+                <Form.Control
+                  required
+                  defaultValue={user.name}
+                  type="text"
+                  placeholder="Įrašykite vardą"
+                />
+              </Form.Group>
+            </Col>
```

Comment:
Ar neitų šitų visų Form.Group išextractint į atskirą functional component? Maždaug taip
```
const FormField = ({ controlId, label, type, placeholder, defaultValue }) => (
  <Form.Group className="mb-3" controlId={controlId}>
    <Form.Label>{label}</Form.Label>
    <Form.Control required type={type} placeholder={placeholder} defaultValue={defaultValue} />
  </Form.Group>
);
```